configuration/concepts/consultation_prenatale.csv

```diff
@@ -0,0 +1,51 @@
+da18a735-7f7d-4f77-a01c-d6b2ecfc36fe,,,Signs and Symptoms Set,Données sur les signes et symptômes,Signs and Symptoms Set,Données sur les signes et symptômes,,,Concept Details,N/A,,,Signs and Symptoms; Other Signs and Symptoms; Duration; Time Units,,,,,,,,
+30a99cd4-90d6-43f3-9d56-47a7f862a73c,,,Examination not done,Examen non effectué,Examination not done,Examen non effectué,,,Misc,N/A,,,,,,,,,,,
+d291cf12-a568-4eec-9f86-9ef55333c3b2,,,Risky Pregnancy,Grossesse à risque,Risky Pregnancy,Grossesse à risque,,,Misc,Coded,,Yes; No,,,,,,,,,
+61cc09dd-5698-4399-bb9e-1bbafe1f0f44,,CIEL:119476,Pregnancy + diabetes,Grossesse + diabète,Pregnancy + diabetes,Grossesse + diabète,,,Misc,N/A,,,,,,,,,,,
+4550dd56-ee8f-4f47-beb4-50ffc448715f,,CIEL:47,Pregnancy + hypertension,Grossesse + hypertension,Pregnancy + hypertension,Grossesse + hypertension,,,Misc,N/A,,,,,,,,,,,
+dc759555-9e47-4ddc-8dc4-1f10024e7248,,,Pregnancy + HIV,Grossesse + VIH,Pregnancy + HIV,Grossesse + VIH,,,Misc,N/A,,,,,,,,,,,
+967b8ea8-6c73-4195-94bb-03651a90c48b,,,Pregnancy + Syphilis,Grossesse + Syphilis,Pregnancy + Syphilis,Grossesse + Syphilis,,,Misc,N/A,,,,,,,,,,,
+815239ab-6321-4d48-86d4-1e6d8f47a94b,,,Pregnancy + obesity,Grossesse + obésité,Pregnancy + obesity,Grossesse + obésité,,,Misc,N/A,,,,,,,,,,,
+9ea362b1-72a2-4b5e-9bde-13572f37de56,,CIEL:157648,Pregnancy + malnutrition,Grossesse + malnutrition,Pregnancy + malnutrition,Grossesse + malnutrition,,,Misc,N/A,,,,,,,,,,,
+50e4eb00-968a-456f-824b-cbe320064dcf,,,Mother younger than 18 or older than 36 years,Mère âgée de moins de 18 ans ou de plus de 36 ans,Mother younger than 18 or older than 36 years,Mère âgée de moins de 18 ans ou de plus de 36 ans,,,Misc,N/A,,,,,,,,,,,
+b35ad290-0d8f-4821-91fe-70b8411ce110,,,Coded Pregnancy Risks,Risques de grossesse codés,Coded Pregnancy Risks,Risques de grossesse codés,,,Misc,Coded,,Multiple gestation (O30.9);Pregnancy + diabetes;Pregnancy + hypertension;Pregnancy + HIV;Pregnancy + Syphilis;Pregnancy + obesity;Pregnancy + malnutrition;Mother younger than 18 or older than 36 years;Other,,,,,,,,,
+ee39e8af-12e8-41ae-a1f9-b08acea07587,,,Pregnancy Risks,Risques de grossesse,Pregnancy Risks,Risques de grossesse,,,Misc,Text,,,,,,,,,,,
+0ae11d29-2ba0-4bae-a2fe-47603a82df3d,,,Childbirth plan,Plan d'accouchement,Childbirth plan,Plan d'accouchement,,,Misc,Coded,,Yes;No,,,,,,,,,
+6ef80ed0-2102-45d3-a188-ad57c16263dc,,,Planned place of delivery,Lieu prévu pour l'accouchement,Planned place of delivery,Lieu prévu pour l'accouchement,,,Misc,Coded,,St Jean Limbe Hospital;Good Samaritain Limbe Hospital;Justinien Cap Haitien Hospital;Sacred Heart Milot Hospital;HBCH Hospital;Gable Hospital;Borgne Hospital;Trou du Nord Hospital;Home;Other,,,,,,,,,
+657e7263-a3d6-445e-9c24-81e765ea447d,,,Planned place of delivery set,Ensemble de lieu de livraison prévu,Planned place of delivery set,Ensemble de lieu de livraison prévu,,,ConvSet,N/A,,,Planned place/location for delivery; Other Health Centre,,,,,,,,
```

Comment:
Let's use "Set" directly as the english term.
"Lieu prévu pour l'accouchement set"

Reply:
Done